default/stringtables/en.txt

```diff
@@ -5957,77 +5957,47 @@ An Experimentor outpost exists in this system, and defeating them is necessary t
 PROBIOTIC_SPECIAL
 Probiotic Soup
 PROBIOTIC_SPECIAL_DESC
-'''The great diversity of unique single-celled organisms have applications to the health and maintenance of all types of organic life.
-
-[[encyclopedia ORGANIC_SPECIES_TITLE]] [[GROWTH_SPECIAL_POPULATION_INCREASE]] [[ORGANIC_SPECIES_TITLE]].'''
+The great diversity of unique single-celled organisms have applications to the health and maintenance of all types of organic life.
```

Comment:
Ah, I see, I was working on a different branch and hadn't compiled this, merely reviewing the changes, I hadn't noticed the macro calling the text within the effectsgroups. Rather cunning and definitely makes thins easier going forward.